src/simplewallet/simplewallet.cpp

```diff
@@ -2307,27 +2229,6 @@ bool simple_wallet::on_cancelled_command()
   return true;
 }
 
-bool simple_wallet::cold_sign_tx(const std::vector<tools::wallet2::pending_tx>& ptx_vector, tools::wallet2::signed_tx_set &exported_txs, std::vector<cryptonote::address_parse_info> &dsts_info, std::function<bool(const tools::wallet2::signed_tx_set &)> accept_func)
```

Comment:
The Wallet API seems to implement cold signing, but I don't see any replacement for this old `cold_sign_tx` method here, which was used in the old code, nor any calls to the Wallet API cold signing method `coldSignTx` in general. What do I overlook?